### ISSUE TITLE
Add bundle_id to apple_resource_bundle rule or support Info.plist variable substitution for PRODUCT_BUNDLE_IDENTIFIER

### ISSUE THREAD

**Issue by thii** [bazelbuild/rules_apple#679]
The `Info.plist` file comes with a resource bundle target out of the box coming from Xcode by default has a `PRODUCT_BUNDLE_IDENTIFIER` variable. Although bundle ID isn't needed for resource bundles (according to Bazel) it would be nice to make this work for the default targets that get generated by Xcode.

**Comment by thii** [comment]
We're currently having to workaround this by editing the generated Info.plist every time we add a new resource bundle target.

**Comment by steeve** [comment]
As a workaround, you can use a script that uses `plutil` like this, for example for the version:
```python
ios_application(
...
    ipa_post_processor = ":plist_generate",
...
)

sh_binary(
    name = "plist_generate",
    srcs = ["plist_generate.sh"],
)
```

```bash
#!/bin/bash

set -euo pipefail

archive_root_path=${1}
framework_path=$(find ${archive_root_path} -type d -name '*.framework')
framework_name=$(basename ${framework_path%.framework*})

# Read the stamps to get the version
eval $(cat bazel-out/stable-status.txt bazel-out/volatile-status.txt | sed 's/ /=/')

plutil -convert binary1 -o "${framework_path}/Info.plist" - <<EOF
{
    "CFBundleExecutable": "${framework_name}",
    "CFBundlePackageType": "FMWK",
    "CFBundleShortVersionString": "${STABLE_VERSION}",
    "MinimumOSVersion": "11.0",
    "CFBundleSupportedPlatforms": [
        "iPhoneOS",
        "iPhoneSimulator",
    ],
}
EOF
```

**Comment by jschear** [comment]
Looks like this has been resolved by #1182.
